src/main/java/Milo/Parser.java

```diff
@@ -0,0 +1,133 @@
+package Milo;
+
+import Milo.TaskObj.*;
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+
+/*
+* Represents Milo's logic system, how the commands are interpreted
+* these commands include: list, mark, unmark, delete, to-do, deadline, event,
+* bye
+ */
+public class Parser {
+    /*
+    * Interprets user input as one of the aforementioned command
+    * subsequently, executes the respective command
+    * given that it is formatted correctly
+    * else an error is thrown and printed
+     */
+    public static void readInput(String userInput, ArrayList<Task> todoList) {
+        String[] arrOfInput = userInput.split(" ", 2);
+        String action = arrOfInput[0];
+        switch (action) {
+            // Show list
+            case "list":
+                Ui.printList(todoList);
+                break;
+            // Find task
+            case "find":
```

Comment:
I like how you use switch statement on the action keyword, but do check that your switch statement follows the coding convention (no indentation for case clauses).